### PR TITLE
STREAMS-1868 explain how to encrypt Kafka data at rest

### PR DESCRIPTION
**Pull request by cremond**
Thank you for your contribution to the Axway Streams documentation.

## Describe the changes

Add a paragraph to explain how to encrypt Kafka data at rest.

## Checklist for contributors

Before submitting this PR, please make sure:

* [x] You have read the [contribution guidelines](https://axway-open-docs.netlify.com/docs/contribution_guidelines/)
* [x] You have signed the [Axway CLA](https://cla.axway.com/)
* [x] You have verified the technical accuracy of your change
* [x] You have verified that your change does not expose any sensitive information (passwords, keys, etc.)
* [x] You have followed the [Markdown guidelines](https://axway-open-docs.netlify.com/docs/contribution_guidelines/writing_markdown/)  (unless this is is a Netlify CMS contribution)

### CODE REVIEW

**Comment by cremond** [comment]
The only change is here: https://deploy-preview-114--streams-open-docs.netlify.app/docs/architecture/#encrypting-volume-data-at-rest

**Comment by jfmarquet** [comment]
Change approved.

**Comment by cremond** [comment]
@andreamussap technical review has been done, you can now review! Thanks in advance :)

File: content/en/docs/architecture/_index.md [hunk 1]
```diff
@@ -103,6 +103,12 @@ The bastion must have high traceability with specific RBAC permissions to allow
 
 Platform infrastructure must support Kubernetes PersistentVolumes. For more information, see [Volumes](#volumes).
 
+#### Encrypting volume data at rest
+
+We recommend you to use an externalized Kafka, managed by a cloud provider, as opposed to an embedded Kafka. But, if you're using an embedded Kafka, you must ensure that the [volumes](#volumes) used are encrypted.
```
Comment (andreamussap) [inline content/en/docs/architecture/_index.md:108]: @cremond I've made some changes. Please let me know if they're ok and I can merge this. Thanks

Comment (cremond) [reply content/en/docs/architecture/_index.md:108]: I've just had a look, it's all good, thanks! (and much more readable ^^)

**Review by andreamussap — APPROVED** [review]

**Pull request merged**